### ISSUE TITLE
Navigate up and down the list with arrows

### ISSUE THREAD

**Issue by kenjinp** [BlueBrain/nexus-explorer#34]
There should be keyboard navigation and a snazzy animation between resource detail views

**Comment by mdupont** [comment]
This issue was moved to BlueBrain/nexus#152
